packages/services/f-services/src/validations.js

```diff
@@ -46,7 +46,7 @@ const isValidPostcode = (postcode, locale) => POSTCODE_REGEX[locale].test(postco
 const meetsCharacterValidationRules = value => /^[\u0060\u00C0-\u00F6\u00F8-\u017Fa-zA-Z-' ]*$/.test(value);
 
 const PHONE_REGEX = {
-    'en-GB': /^(\+(44))?[0-9]{10}|(\+(44))?[0-9]{11}$/,
+    'en-GB': /^(\+(44))?[0-9]{10}|^(\+(44))?[0-9]{11}/,
```

Comment:
Does this require any new test cases?

Reply:
No

Reply:
What's the difference between the 2 regex's, if we don't need a test to cover what was wrong with the initial pattern?

Reply:
The replacement doesn't look right to me either - if I'm reading it correctly it will expect any string as long as it's prefixed by something resembling a valid phone number, where before it would accept either a string beginning with a number of 10 digits or a string ending with a number of 11 digits. I'd personally go for something like:
```suggestion
    'en-GB': /^(\+(44))?[0-9]{10,11}$/,
```
to avoid the repetition.